### DISCUSSION TITLE
Setup Help

### DISCUSSION THREAD

**Discussion by dtownsend43** [DMcP89/harambot#43] [Q&A] [answered]
New to this and would like to implement this into my Discord server but I am stuck on the "make configure" part using git bash. Could you please help

**Comment by DMcP89** [comment]
So I would recommend using the windows subsystem for linux (WSL) over Git Bash on a windows machine. All of the development work for the bot was done in WSL so you should have no problems there.

https://docs.microsoft.com/en-us/learn/modules/get-started-with-windows-subsystem-for-linux/

However if you're dead set on using Git Bash you can follow this guide to install make.

https://gist.github.com/evanwill/0207876c3243bbb6863e65ec5dc3f058

**Comment by dtownsend43** [comment, marked as answer]
Ok thank you!

**Reply by DMcP89** [reply]
If this has helped resolve your issue can you mark this question as answered?
